pyflow/scene/clipboard.py

```diff
@@ -3,45 +3,42 @@
 
 """ Module for the handling of scene clipboard operations. """
 
-from typing import TYPE_CHECKING, OrderedDict
+from typing import TYPE_CHECKING, OrderedDict, Union
 from warnings import warn
 
-import json
-from PyQt5.QtWidgets import QApplication
-
 from pyflow.core.edge import Edge
 
 if TYPE_CHECKING:
     from pyflow.scene import Scene
-    from pyflow.graphics.view import View
-
-
-class SceneClipboard:
 
-    """Helper object to handle clipboard operations on an Scene."""
 
-    def __init__(self, scene: "Scene"):
-        """Helper object to handle clipboard operations on an Scene.
+class BlocksClipboard:
 
-        Args:
-            scene: Scene reference.
+    """Helper object to handle clipboard operations on blocks."""
 
-        """
-        self.scene = scene
+    def __init__(self):
+        """Helper object to handle clipboard operations on blocks."""
+        self.blocks_data: Union[None, OrderedDict] = None
 
-    def cut(self):
+    def cut(self, scene: "Scene"):
         """Cut the selected items and put them into clipboard."""
+        self.scene = scene
```

Comment:
The attribute scene should be set in the __init__ function, or not be an attribute at all.